### PR TITLE
Add ability to rename a project

### PR DESCRIPTION
**Pull request by dzervas**

### CODE REVIEW

**Comment by av8ta** [comment]
Works great, thanks! Just a couple minor nits noted in the comments and can you rebase off main?

**Comment by dzervas** [comment]
@av8ta I don't see any other comments

File: applications/web/src/routes/(CADmium)/AppBar.svelte [hunk 1]
```diff
@@ -46,7 +49,35 @@
...
+					log("Renaming project")
+					renameProject(newProjectName)
+					project.name = newProjectName
+					renaming = false
+				}
+			}}
+		/>
+		{:else}
+		<div
+			class="text-xl font-medium"
```
Comment (av8ta) [inline applications/web/src/routes/(CADmium)/AppBar.svelte:73]: Please remove `text-xl`

**Comment by av8ta** [comment]
> @av8ta I don't see any other comments

The comments in the review above.

**Pull request merged**